### DISCUSSION TITLE
Integration with Lenovo Legion GO

### DISCUSSION THREAD

**Discussion by drunksnake** [Valkirie/HandheldCompanion#856] [Ideas]
Hi,

I am very hoping there is a version of Handheld Companion for the Lenovo Legion GO. Anyone know if it's something coming soon ?

Thank you,

**Comment by virajmalia** [comment]
I checked the discord announcement that the 0.19 build has support for Legion Go. It is available to Patreon subscribers since December 04, 2023.

**Reply by drunksnake** [reply]
Great news... Thanks for the info!

**Comment by CasperH2O** [comment]
Indeed, still needs Joy con gyro and fan support.

**Comment by CasperH2O** [comment]
Duplicate of https://github.com/Valkirie/HandheldCompanion/issues/816, closing.
